### DISCUSSION TITLE
Default font family name can't be null or empty.

### DISCUSSION THREAD

**Discussion by Schoki100g** [AvaloniaUI/Avalonia#9653] [General]
Hi, 

I have tried to run the Multi-platform template on Arch Linux and this error show up while starting the app.
Can someone help me fix this error?

**Comment by timunie** [comment]
A bit more context would be useful. 

1. Which version of Avalonia?
2. What have you changed so far?
3. If possible some screenshots
4. If possible a sample

Also, maybe Arch Linux uses Wayland. If so, we have a PR for adding better Wayland support. Feel free to test it.

**Comment by maxkatz6** [comment]
Please follow this issue https://github.com/AvaloniaUI/Avalonia/issues/4427
